frontend/src/components/CandidateInputComponent.module.css

```diff
@@ -0,0 +1,31 @@
+.inputBody {
+  width: 100%;
+  background-color: #d9d9d9;
+  display: flex;
+  gap: 20px;
+  border-radius: 40px;
+  padding: 7px 20px;
+}
+.inputBody p {
+  font-size: 24px;
+  text-transform: uppercase;
+  font-weight: 400;
+}
+
+.radioBtn {
+  width: 2rem;
+  height: 2rem;
+  border: solid black;
+  border-radius: 50%;
```

Comment:
if you want to make something have rounded borders, set border-radius in pixels. You can even set it to 9999px, it's going to just be rounded borders. Setting 50% works here because width is the same as height, but otherwise, it wouldn't work well.